cpp/velox/substrait/SubstraitToVeloxPlanValidator.cc

```diff
@@ -469,9 +469,13 @@ bool SubstraitToVeloxPlanValidator::validate(const ::substrait::GenerateRel& gen
     names.emplace_back(SubstraitParser::makeNodeName(inputPlanNodeId, colIdx));
   }
   auto rowType = std::make_shared<RowType>(std::move(names), std::move(types));
-
-  if (generateRel.has_generator() && !validateExpression(generateRel.generator(), rowType)) {
-    LOG_VALIDATION_MSG("Input validation fails in GenerateRel.");
+  try {
+    if (generateRel.has_generator() && !validateExpression(generateRel.generator(), rowType)) {
+      LOG_VALIDATION_MSG("Input validation fails in GenerateRel.");
+      return false;
+    }
+  } catch (const VeloxException& err) {
```

Comment:
we may add try catch block at https://github.com/apache/incubator-gluten/blob/8bf9a7bb1a1c796553ad42422f30cb8d01ca1197/cpp/velox/substrait/SubstraitToVeloxPlanValidator.cc#L1260-L1274 for all expected exceptions, thus Fallback reporter could catch it.

Reply:
Good idea, I think then we can remove other silmiar try catch in this file.

Reply:
> I think then we can remove other silmiar try catch in this file.

yeah, we may need remove all try catch under `validate` entrances, avoid such exceptions be caught/wrapped, and helps find root fallback reason/exception.